web/src/lib/components/admin-page/settings/setting-switch.svelte

```diff
@@ -7,6 +7,13 @@
   export let checked = false;
   export let disabled = false;
   export let isEdited = false;
+  export let onToggle: (checked: boolean) => void = () => {
+    return;
+  };
+  export const handler = (e: Event) => {
+    const target = e.target as HTMLInputElement;
+    onToggle(target.checked);
+  };
```

Comment:
This is exactly what event dispatchers are for

Reply:
I knew there had to be a better way 😛